tests/Zend/Db/Select/TestCommon.php

```diff
@@ -363,7 +363,12 @@ public function testSelectColumnWithColonQuotedParameter()
         $stmt = $select = $this->_selectColumnWithColonQuotedParameter()
             ->query();
         $result = $stmt->fetchAll();
-        $this->assertEquals(0, count($result));
+
+        if (is_array($result)) {
```

Comment:
The `fetchAll` method above can return `null` or an empty array, testing for both conditions.